fastlane/lanes/localization.rb

```diff
@@ -134,8 +134,7 @@
 
     push_to_git_remote(tags: false)
 
-    release_branch = "release/#{version}"
-    create_release_management_pull_request(release_branch, "Merge #{version} editorialized release notes to #{release_branch}")
+    create_backmerge_pr
```

Comment:
@iangmaia @mokagio Unfortunately this change is not correct. The previous behavior was to create a PR to merge the `release_notes/{version}` branch to `release/{version}` branch whereas the new behavior is to merge `release/{version}` to `trunk` with an intermediate branch.

For example, the [build](https://buildkite.com/automattic/wordpress-android/builds/20002#01928c97-ccfd-4daa-ba6f-d7b782079e08) that's supposed to update the release notes didn't open the correct PR. There are a couple more issues in that build:
1. The build was successful even though `fastlane` was unable to open the PR. If it did it wouldn't be the correct PR, but regardless I think it should fail the build when an error occurs.
2. The build is named `Code Freeze` even though it's updating release notes.

Note that for `25.4` release, I'll handle the issue manually, so there is no rush to fix this. However, please do let me know that you've seen this comment so I know that it won't go unnoticed. Thank you! 🙇‍♂️

Reply:
Thanks @oguzkocer, and sorry again for the trouble.

We've been experimenting with slightly simplified workflows but indeed I see some mismatches between what's in ReleasesV2 and the automation here.
I think the intention was to replace the 2 distinct steps ( **a**. `release_notes/{version}` merge to the release branch + **b**. create a backmerge PR to `trunk`) into a single backmerge with an intermediate branch, but that is currently is not correct as you highlighted.

The `Code Freeze` build name is probably to old data in ReleasesV2 (we should have reset the existing releases after we updated it).

I'll look into it (if @mokagio doesn't beat me to it), hopefully still this week.

Again, thanks a lot!

Reply:
For the record, this PR should fix this issue: https://github.com/wordpress-mobile/WordPress-Android/pull/21397